Project/contracts/Ballot.sol

```diff
@@ -2,61 +2,63 @@
 pragma solidity >=0.7.0 <0.9.0;
 
 /// @title Voting with delegation.
-contract Ballot {
-    // This declares a new complex type which will
-    // be used for variables later.
-    // It will represent a single voter.
+/// @author The Buidlers
+/// @notice You can use this contract for understanding how voting contracts work
+/// @dev You're gonna make it
+contract Ballot {    
+    /**
+     * @dev This declares a new complex type which will be used for variables later. It represents a single voter.    
+     * - voted: if true, that person already voted
+     * - delegate: user delegated to
+     * - weight: accumulated by delegation
+     * - vote: index of the voted proposal
+     */
     struct Voter {
-        bool voted; // if true, that person already voted
-        address delegate; // person delegated to
-        uint256 weight; // weight is accumulated by delegation
-        uint256 vote; // index of the voted proposal
```

Comment:
You could have added` /// @dev` annotations to these struct members instead of deleting comments that were not respecting NatSpec format.

Reply:
I moved them to `/// @dev` above the struct, check lines 12-15.